Content.Client/Shuttles/UI/BaseShuttleControl.xaml.cs

```diff
@@ -115,6 +115,15 @@ protected void DrawCircles(DrawingHandleScreen handle)
         }
     }
 
+    // Frontier Corvax: north line drawing
+    protected void DrawNorthLine(DrawingHandleScreen handle, Angle angle)
+    {
+        var origin = ScalePosition(-new Vector2(Offset.X, -Offset.Y));
+        var aExtent = (angle - Math.Tau / 4).ToVec() * ScaledMinimapRadius * 1.42f;
+        var lineColor = Color.Red.WithAlpha(0.02f);
```

Comment:
```suggestion
        var lineColor = Color.Red.WithAlpha(0.05f);
```

Might be my monitor setup, but the line can be difficult to make out at 2% opacity, especially in poor lighting.  5% seems more reasonable.

Below: 2% opacity (as-is).
![image](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/2de3bf34-eb4f-4af5-ac70-8ad545835acd)

Below: 5% opacity (suggested).
![image](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/1f78f6de-260d-4c1a-800d-e75139ade0c8)

Reply:
I would have to agree, my eye sight is bad enough already that the current lines are difficult for me. Everything else is good. I'd even be fine with going all the way up to 10%